<launch>

  <!-- params -->
  <arg name="robot_ip" default="192.168.1.214"/>
  <param name="xarm_robot_ip" type="string" value="192.168.1.196"/>
  <arg name="joy_dev" default="/dev/input/js0" />
  <arg name="joystick" default="true" />

  <!--joystick node-->
  <node name="joystick_node" pkg="joy" type="joy_node"></node>

  <!-- xarm launch -->
  <include file="$(find xarm_bringup)/launch/xarm6_server.launch">
      <arg name="robot_ip" value="$(arg robot_ip)"/>
      <arg name="show_rviz" value="true"/>
  </include>

  <!-- <param name="my_robot_description" textfile="$(find xarm_description)/urdf/xarm6.urdf.xacro"/> -->
    
  <!-- <node pkg="xarm_api" type="xarm_driver_node" name="xarm_driver_node" >
      <remap from="robot_description" to="my_robot_description" />
      <remap from="joint_states" to="joint_states" />
  </node> -->

</launch>